src/components/Navbar/SearchBar.jsx

```diff
@@ -0,0 +1,35 @@
+import {useState} from "react";
+import { useNavigate } from "react-router-dom";
+import {ReactComponent as SearchIcon} from "../../assets/Search.svg"
+
+const SearchBar=()=>{
+    
+
+    const [searchParam,setSearchParam]=useState("");
+    const navigate = useNavigate();
+
+    
+    const onSearchChangeHandler=(e)=>{
+        setSearchParam(e.target.value)
+          
+    }
```

Comment:
Suggestion : Use Prettier Extension in VS Code to auto format code upon Saving File